tests/apiUnit.js

```diff
@@ -0,0 +1,204 @@
+          a: 'value A',
+          b: 'value B',
+          updated: {} };
+
+        return digger.setClient(5)
+        .then(() => engine.saveData([testData], 5))
+        .then(() => digger.getFeatureStatus('feature', 5))
+        .then(featureStatus => {
+          assert.ok(featureStatus);
+          assert.deepEqual(featureStatus, testData);
+        });
+      });
+    });
+  });
+
+  bdd.describe('api route', () => {
+    const portfinder = require('intern/dojo/node!portfinder');
+    const platatus = require('intern/dojo/node!../app');
+
+    var port;
+    var server;
+    function api() {
+      return chai.request(`http://localhost:${port}`);
+    }
+
+    // set database channel
+    bdd.beforeEach(() => digger.setClient(5));
+
+    // find port and spin the server
+    bdd.before(() => new Promise((resolve) => {
+      portfinder.getPort((err, receivedPort) => {
+        port = receivedPort;
+        server = platatus.listen(port, () => {
+          console.log('server is running on port %j', port);
+          resolve();
+        });
+      });
+    }));
+
+    bdd.after(() => server.close());
+
+    // clean and quit database after each test
+    bdd.afterEach(() =>
+      digger.setClient(5)
+      .then((client) => flushQuitDB(client)));
+
+    bdd.describe('/api/status', () => {
+      bdd.it('returns null if no status', () =>
+        api()
+        .get('/api/status')
+        .send()
+        .then(response => {
+          assert.strictEqual(response.body, null);
+        })
+      );
+
+      bdd.it('should return the value stored under key defined by slug', () => {
+        const testData = {
+          slug: 'feature',
+          a: 'value A',
+          b: 'value B',
+          updated: {} };
+
+        return digger.setClient(5)
+        .then(() => engine.saveData([testData], 5))
+        .then(() => api()
+          .get('/api/status')
+          .send()
+        )
+        .then(response => {
+          assert.property(response.body, 'feature');
+          assert.deepEqual(response.body.feature, testData);
+        });
+      });
+    });
+
+    bdd.describe('/api/feature/{slug}', () => {
+      bdd.it('returns 404 if getFeatureStatus throws', () =>
+        api()
+        .get('/api/feature/some-status')
+        .send()
+        .catch(err => {
+          assert.equal(err.response.status, 404);
+        })
+      );
+
+      bdd.it('returns savedData', () => {
+        const testData1 = {
+          slug: 'feature',
+          a: 'value A',
+          b: 'value B',
+          updated: {} };
+        const testData2 = {
+          slug: 'another-feature',
+          a: 'another A',
+          b: 'another B',
+          updated: {} };
+
+        return digger.setClient(5)
+        .then(() => engine.saveData([testData1, testData2], 5))
+        .then(() => api()
+          .get('/api/feature/feature')
+          .send()
+        )
+        .then(response => {
+          assert.property(response.body, 'slug');
```

Comment:
No need, you're using `deepEqual` later. So I guess you can remove this check then.